xbee_at_command_block.py

```diff
@@ -1,10 +1,14 @@
 import binascii
 from nio.util.discovery import discoverable
-from nio.properties import Property
+from nio.properties import Property, ListProperty, PropertyHolder
 from nio.properties.version import VersionProperty
 from .xbee_base import XBeeBase
 
 
+class ATCommand(PropertyHolder):
+    command   = Property(title='AT Command Name (ascii)', default='ID')
```

Comment:
Take a look at PEP-8 and update the code accordingly. Remove additional spaces here. Also rename variable to not use camel case.

https://www.python.org/dev/peps/pep-0008/